src/init.lua

```diff
@@ -1797,23 +1792,24 @@ xplr.config.modes.builtin.delete = {
         help = "delete",
         messages = {
           {
-            BashExec = [===[
-              (while IFS= read -r line; do
-              if [ -d "$line" ] && [ ! -L "$line" ]; then
-                if rmdir -v -- "${line:?}"; then
-                  echo LogSuccess: "'$line deleted'" >> "${XPLR_PIPE_MSG_IN:?}"
-                else
-                  echo LogError: "'Failed to delete $line'" >> "${XPLR_PIPE_MSG_IN:?}"
-                fi
-              else
-                if rm -v -- "${line:?}"; then
-                  echo LogSuccess: "'$line deleted'" >> "${XPLR_PIPE_MSG_IN:?}"
+            BashExec0 = [===[
+              (while IFS= read -r -d '' LINE; do
+                LINE_ESC=${LINE//\"/\\\"}
+                if [ -d "$LINE" ] && [ ! -L "$LINE" ]; then
+                  if rmdir -v -- "${LINE:?}"; then
+                    printf 'LogSuccess: "%s"\0' "$LINE_ESC deleted" >> "${XPLR_PIPE_MSG_IN:?}"
```

Comment:
Regarding my earlier comment about the logs pipe, this might need to be:
```sh
printf 'LogSuccess: "%q deleted"\0' "$LINE_ESC" ...
```

What do you think?

Reply:
'%q' will unnecessarily escape single quotes and spaces too, which might be confusing to the user.

Reply:
Actually, thinking more on this, it might be a good idea to use shell escape in the logs. So, we can copy paste the paths in the terminal to run any command.

Reply:
In shell scripts, I always use `%q` when logging paths, because it doesn't break up the log line and shows exactly what the string's content is, and it definitely makes copy/pasting the paths nicer.

In this case, since the latest log line is a single line, it may make that easier to read, too.

Reply:
Actually, the way YAML is encoded, `%q` is problematic with filenames like `a 'b "c`, where `%s` works fine.

Reply:
Added subcommand `-m`, `--pipe-msg-in` to help with all the YAML escaping. It will allow `%s`, `%q` and `%%` placeholders similar to `printf`, but for YAML instead.